install_files/securedrop-app-code/DEBIAN/postinst

```diff
@@ -55,6 +55,13 @@ case "$1" in
       rm /tmp/securedrop_custom_logo.png
     fi
 
+    # Repoint tor repositories to FPF mirror
+    for file in "sources.list.d/deb_torproject_org_torproject_org.list" "security.list"
+    do
+        if [ -f "/etc/apt/$file" ]; then
+            sed -i 's/deb.torproject.org\/torproject.org/tor-apt.ops.freedom.press/g' "/etc/apt/$file"
+        fi
+    done
```

Comment:
This strategy will only update the apt config on the App Server, not on the Mon Server. Definitely it's critical that we minimize breakage for the App Server interfaces, but we need a way to land config changes across the board. Debian configuration packages come to mind: https://debathena.mit.edu/config-packages/